app/validators/blocks/list_collector_validator.py

```diff
@@ -104,16 +109,23 @@ def validate_other_list_collectors(self):
                     list_name=list_name,
                 )
 
-    def validate_single_repeating_blocks_list_collector(self):
+    def validate_repeating_blocks_list_collectors(self, same_type, other_type):
         if self.block.get("repeating_blocks"):
             list_name = self.block["for_list"]
 
-            other_list_collectors = self.questionnaire_schema.get_other_blocks(
-                self.block["id"], for_list=list_name, type="ListCollector"
+            list_collectors_other_type = self.questionnaire_schema.get_other_blocks(
+                self.block["id"], for_list=list_name, type=other_type
+            )
+            other_list_collectors_same_type = (
+                self.questionnaire_schema.get_other_blocks(
+                    self.block["id"], for_list=list_name, type=same_type
+                )
             )
 
-            if other_list_collectors:
+            if (
+                list_collectors_other_type and len(list_collectors_other_type) > 1
```

Comment:
minor non-blocker `get_other_blocks` always returns a list, so this could just check `len(list_collectors_other_type) > 1`

Reply:
wait, actually, why does it need to check its bigger than 1? Is it OK to have a ListCollector and ListCollectorContent for the same list both with repeating blocks?

Reply:
That should work fine.

Reply:
in that case - all good from me